src/Api/Controllers/OrganizationsController.cs

```diff
@@ -163,10 +163,17 @@ public async Task<OrganizationResponseModel> Post([FromBody]OrganizationCreateRe
             }
 
             var policies = await _policyRepository.GetManyByUserIdAsync(user.Id);
-            if (policies.Any(policy => policy.Enabled && policy.Type == PolicyType.SingleOrg))
+            foreach (var policy in policies)
             {
-                throw new Exception("You may not create an organization. You belong to an organization " +
-                     "which has a policy that prohibits you from being a member of any other organization.");
+                if (policy.Enabled && policy.Type == PolicyType.SingleOrg)
+                {
+                    var organizationUser = await _organizationUserRepository.GetByOrganizationAsync(policy.OrganizationId, user.Id);
```

Comment:
This is going to be expensive in a loop construct, especially for a user with many orgs that have this policy -> I would instead pull all user organizations once before the loop (and then only do that if your list of policies, say that `.Where(p =>)` lambda LINQ query into a variable and only if that variable.Any() pull all user orgs for that user, then test within the loop. Something like that. Just don't want 1:1 DB query:Policy